DEVELOPMENT.md

```diff
@@ -0,0 +1,105 @@
+# Developing
+
+## Getting started
+
+Many of the instructions here replicate what's in the [tektoncd/pipeline development guide](https://github.com/tektoncd/pipeline/blob/master/DEVELOPMENT.md), with a couple of caveats currently.
+
+1. This project currently does not support being built with `ko`
```

Comment:
Out of curiosity, what doesn't work with `ko` ?

Reply:
I'll be upfront - we haven't tried it for this project (only for knative repositories and tektoncd/pipeline), I had a look around the pipeline repo and I noticed lots of `kodata` folders so I'm unsure about using it just yet, perhaps we need it? Would be awesome if it all worked!